src/test/java/api/TenantActivationResourceTests.java

```diff
@@ -68,27 +67,12 @@ public void tenantActivationSucceedsWhenCanRegisterInPubSub() {
   }
 
   @Test
-  public void tenantDeactivationFailsWhenCannotUnregisterWithPubSub() {
+  public void tenantDeactivationSucceedsWhenCannotUnregisterInPubSub() {
     setFailPubSubUnregistering(true);
 
-    Response response = tenantActivationFixture.deleteTenant();
-    assertThat(response.getStatusCode(), is(HTTP_INTERNAL_SERVER_ERROR.toInt()));
-  }
-
-  @Test
-  public void tenantDeactivationSucceedsWhenCanUnregisterInPubSub() {
     Response response = tenantActivationFixture.deleteTenant();
 
     assertThat(response.getStatusCode(), is(HTTP_NO_CONTENT.toInt()));
-
-    assertThat(getDeletedEventTypes().size(), is(6));
-    assertThat(getDeletedEventTypes(), hasItems(
-      EventTypeMatchers.ITEM_CHECKED_OUT,
-      EventTypeMatchers.ITEM_CHECKED_IN,
-      EventTypeMatchers.ITEM_DECLARED_LOST,
-      EventTypeMatchers.ITEM_CLAIMED_RETURNED,
-      EventTypeMatchers.LOAN_DUE_DATE_CHANGED,
-      EventTypeMatchers.LOG_RECORD
-    ));
+    assertThat(getDeletedEventTypes().size(), is(0));
```

Comment:
Would it be better to assert that pubsub was not called? With PowerMock, for example.

Reply:
PubSub is mocked by FakePubSub and we're checking here if we tried to delete any of our subscriptions/publications from this mocked service. Do you think it's substantially different?

Reply:
I just looked at what FakePubsub does when we try to delete tenant, and you are right, it basically produces the same result I would have hoped to achieve with PowerMock.